docs/attributes-registry/cicd.md

```diff
@@ -28,3 +43,21 @@ This group describes attributes specific to pipelines within a Continuous Integr
 | `build` | build | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `deploy` | deploy | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `test` | test | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+---
+
+`cicd.worker.state` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
```

Comment:
I think we should add `down` or `disconnected`

Reply:
Added in 7330043f03834f396be3a0d8800d1473ec63784a